src/Core/Geographic/GeoCoordinate.js

```diff
@@ -0,0 +1,204 @@
+/**
+ * Generated On: 2015-10-5
+ * Class: GeoCoordinate
+ * Description: Coordonnées cartographiques
+ */
+/**
+ *
+ * @param {type} defaultValue
+ * @returns {CoordCarto_L9.GeoCoordinate}
+ */
+import defaultValue from 'Core/defaultValue';
+import mE from 'Core/Math/MathExtented';
+
+/* eslint-disable */
+var UNIT_RADIAN = 0;
```

Comment:
Use `const` instead of `var`?
Also I don't see the point of disabling the linter. Remove `COORD_ALT` instead if it's not used.